tests/integration/targets/azure_rm_openshiftmanagedcluster/tasks/main.yml

```
@@ -106,6 +106,47 @@
 - assert:
       that: output['clusters'] | length >= 1
 
+- name: Fetch kubeconfig file to register
+  azure_rm_openshiftmanagedclusterkubeconfig_info:
+    name: "{{ cluster_name }}"
+    resource_group: "{{ resource_group }}"
+  register: kubeconfig_reg
+
+- assert:
```

Use `ansible.builtin.assert` instead of ‘assert' and all tasks should be named. The standard format for Playbooks is to indent two Spaces, so please strictly follow this. Thank you!
```suggestion
- name: Assert the kubeconfig file facts
  assert:
```

---

Done